client/my-sites/customer-home/cards/launchpad/keep-building.tsx

```diff
@@ -34,13 +34,29 @@ const LaunchpadKeepBuilding = ( { siteSlug }: LaunchpadKeepBuildingProps ): JSX.
 	const numberOfSteps = checklist?.length || 0;
 	const completedSteps = ( checklist?.filter( ( task: Task ) => task.completed ) || [] ).length;
 
+	recordTracksEvent( 'calypso_launchpad_tasklist_viewed', {
+		checklist_slug: checklistSlug,
+		tasks: `,${ checklist?.map( ( task: Task ) => task.id ).join( ',' ) },`,
```

Comment:
Is this comma at the beginning a typo?

<img width="1309" alt="Screen Shot 2023-06-21 at 11 59 31 AM" src="https://github.com/Automattic/wp-calypso/assets/2124984/baeb9a71-669d-4771-8f10-781f3c0b7e3b">

Reply:
@markbiek It was a tip from Dale to make our life easier when creating the funnels. Example: if we want to see if the `site_title` task id is present, we just need to check if the `tasks` string contains `,site_title,`, and we don't have to worry about It being the first or last item on the list.

Reply:
Oh interesting! In that case, approved. :D

Reply:
Great trick, thanks for the details!